services/galley/src/Galley/API/Teams.hs

```diff
@@ -783,58 +784,40 @@ updateTeamMember ::
        TeamStore
      ]
     r =>
-  Local UserId ->
-  ConnId ->
+  Maybe (Local UserId) ->
+  Maybe ConnId ->
   TeamId ->
   NewTeamMember ->
   Sem r ()
-updateTeamMember lzusr zcon tid newMember = do
-  let zusr = tUnqualified lzusr
+uncheckedUpdateTeamMember mlzusr mZcon tid newMember = do
+  let mZusr = tUnqualified <$> mlzusr
   let targetMember = ntmNewTeamMember newMember
   let targetId = targetMember ^. userId
       targetPermissions = targetMember ^. permissions
   P.debug $
     Log.field "targets" (toByteString targetId)
-      . Log.field "action" (Log.val "Teams.updateTeamMember")
+      . Log.field "action" (Log.val "Teams.uncheckedUpdateTeamMember")
```

Comment:
i would leave the old value.  they are both equally wrong, but the old one is less misleading when you expect it to be checked, and it actually is called through `updateTeamMember`.